src/sentry/static/sentry/app/views/organizationDiscover/result/utils.jsx

```diff
@@ -41,4 +41,35 @@ const DarkGray = styled.span`
   color: ${p => p.theme.gray5};
 `;
 
-export {getDisplayValue};
+function escapeCol(col) {
+  if (typeof col === 'number') {
+    return col;
+  } else {
+    return `${col}`.match(/^[\=\+\-\@]/) ? `"'${col}'"` : `"${col}"`;
```

Comment:
Should booleans be converted to their stringy values (`'true'` & `'false'`). I think you also need to escape values that contain `"` and `\` in them.

Reply:
I'm tempted to say it would be nice to just pull in a parsing library

Reply:
Yeah, I think i'm going to reimplement this using https://github.com/mholt/PapaParse